### PR TITLE
providing script for plato analysis

### PR DESCRIPTION
**Pull request by daward**

### CODE REVIEW

**Comment by daward** [comment]
When you use this script, it will produce an artifact in S3 that looks like: https://s3-eu-west-1.amazonaws.com/buildartifacts.fi.cimpress.io/s3sync/plato/index.html

File: belt-node-complexity-analysis-ci.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  allow_failure: true
+  script:
+    - npm config set @cimpress-technology:registry=https://artifactory.cimpress.io/artifactory/api/npm/npm-release-local/
+    - npm i @cimpress-technology/s3sync -g
+    - npm i plato -g
+    - s3sync -b ${BUILD_ARTIFACT_BUCKET} -l plato -p ${CI_PROJECT_NAME}/plato -a download
+    - plato -l .eslintrc.json  -r -d plato src
+    - s3sync -b ${BUILD_ARTIFACT_BUCKET} -l plato -p ${CI_PROJECT_NAME}/plato -a upload
+  only:
+    - master
```
Comment (zmarois) [inline belt-node-complexity-analysis-ci.yml:12]: I'm surprised to see this only on master. What is your reasoning?

**Review by zmarois — APPROVED** [review]

File: belt-node-complexity-analysis-ci.yml (hunk 1 above)
Comment (daward) [reply belt-node-complexity-analysis-ci.yml:12]: Being conservative right now, but also, plato works by all publishing to one historical file (not a series of directories), so every MR might make it look rather bouncy historically. Does that make sense?

Comment (zmarois) [reply belt-node-complexity-analysis-ci.yml:12]: Yes.

**Review by acchaulk — APPROVED** [review]

**Pull request merged**